build.gradle

```
@@ -18,11 +10,23 @@ repositories {
   flatDir { dirs 'lib' }
 }
 
-apply plugin: 'java'
-apply plugin: 'org.jetbrains.intellij'
+//apply plugin: 'java'
+//apply plugin: 'org.jetbrains.intellij'
 
 dependencies {
+  // https://mvnrepository.com/artifact/com.googlecode.owasp-java-html-sanitizer/owasp-java-html-sanitizer
+  compile group: 'com.googlecode.owasp-java-html-sanitizer', name: 'owasp-java-html-sanitizer'
   compile "org.jetbrains:markdown:${markdownParserVersion}"
+
+  testCompile group: 'junit', name: 'junit', version: '4.12'
+}
+
+
+compileKotlin {
+  kotlinOptions.jvmTarget = "1.8"
```

在Intellij 里面 跑测试的时候，需要指定这个版本。要不然编译报错
kotlinOptions.apiVersion = "1.3"
```suggestion
  kotlinOptions.jvmTarget = "1.8"
  kotlinOptions.apiVersion = "1.3"
```